deny.toml

```diff
@@ -19,30 +19,30 @@
 allow = [
   "Apache-2.0",
   "Apache-2.0 WITH LLVM-exception",
-  "MIT",
-  "MPL-2.0",
+  "BSD-2-Clause",
   "BSD-3-Clause",
-  "ISC",
   "CC0-1.0",
+  "CDLA-Permissive-2.0",
+  "ISC",
+  "MIT",
+  "MPL-2.0",
+  "MIT",
+  "MPL-2.0",
+  "OpenSSL",
+  "Unicode-3.0",
+  "Unicode-DFS-2016",
+  "Zlib",
 ]
+
 exceptions = [
-  { allow = [
-    "OpenSSL",
-  ], name = "ring" },
-  { allow = [
-    "Unicode-DFS-2016",
-  ], name = "unicode-ident" },
+  # malachite and its subcrates are introduced in the Python bindings (pyo3-stub-gen)
+  # they are only used in proc-macro so they won't be included in the final library
+  { crate = "malachite", allow = ["LGPL-3.0"] },
+  { crate = "malachite-base", allow = ["LGPL-3.0"] },
+  { crate = "malachite-bigint", allow = ["LGPL-3.0"] },
+  { crate = "malachite-nz", allow = ["LGPL-3.0"] },
+  { crate = "malachite-q", allow = ["LGPL-3.0"] },
```

Comment:
May be excluded once https://github.com/Jij-Inc/pyo3-stub-gen/pull/362 gets merged and released.